### DISCUSSION TITLE
HistoryGraph

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#42] [Q&A]
Hey,

gibt es eine Möglichkeit den HistoryGraph im Jarvis so zusammenzuführen dass er nur Tage anzeigt? Ich habe ein Stromzähler und will mir den tägl. Energieverbrauch (kWh) anzeigen lassen. Der History Adapter zeichnet aber ja mehr Werte am Tag auf? Wie kann ich also den "letzten" "höchsten" Wert am Tag in den Graph hauen?

VG

**Comment by mcuiobroker** [comment]
Finde keine Möglichkeit.

**Reply by Mephisto090** [reply]
Also haue ich es mal ins Feature
